homeassistant/components/jvc_projector/strings.json

```diff
@@ -57,6 +115,127 @@
           "cooling": "Cooling",
           "error": "Error"
         }
+      },
+      "jvc_hdmi_input": {
+        "name": "HDMI input",
+        "state": {
+          "hdmi1": "HDMI 1",
+          "hdmi2": "HDMI 2"
+        }
+      },
+      "jvc_picture_mode": {
+        "name": "Picture mode"
+      },
+      "jvc_installation_mode": {
+        "name": "Installation mode"
+      },
+      "jvc_anamorphic_mode": {
+        "name": "Anamorphic mode",
+        "state": {
+          "off": "[%key:common::state::off%]",
+          "a": "Mode A",
+          "b": "Mode B",
+          "c": "Mode C",
+          "d": "Mode D"
+        }
+      },
+      "jvc_hdr_mode": {
+        "name": "HDR mode",
+        "state": {
+          "none": "None",
+          "hdr10": "HDR10",
+          "hdr10_plus": "HDR10+",
+          "hlg": "HLG",
+          "sdr": "SDR"
+        }
+      },
+      "jvc_hdmi_input_level": {
+        "name": "HDMI input level",
+        "state": {
+          "standard": "Standard",
+          "enhanced": "Enhanced",
+          "super_white": "Super White",
+          "auto": "Auto"
+        }
+      },
+      "jvc_hdmi_color_space": {
+        "name": "HDMI color space"
+      },
+      "jvc_color_profile": {
+        "name": "Color profile"
+      },
+      "jvc_graphics_mode": {
+        "name": "Graphics mode",
+        "state": {
+          "standard": "Standard",
+          "high-res": "High Resolution"
+        }
+      },
+      "jvc_color_space": {
+        "name": "Color space",
+        "state": {
+          "rgb": "RGB",
+          "yuv": "YUV"
+        }
+      },
+      "jvc_laser_dimming_mode": {
+        "name": "Laser dimming",
+        "state": {
+          "off": "[%key:common::state::off%]",
+          "auto1": "Auto 1",
+          "auto2": "Auto 2",
+          "auto3": "Auto 3"
+        }
+      },
+      "jvc_laser_value": {
+        "name": "Laser value"
+      },
+      "jvc_laser_power": {
+        "name": "Laser power",
+        "state": {
+          "low": "Low",
+          "medium": "Medium",
+          "high": "High"
+        }
+      },
+      "jvc_laser_time": {
+        "name": "Laser time"
+      },
```

Comment:
I have a lamp, not laser. Could we maybe call this Laser/Lamp? I guess ideally it would detect what to call it based on Model, but a lot of work I know. So maybe just more generic naming is the way to go.

Reply:
done